# target_memory:
#     social_perception_name
#         events: ['event1', 'event2', ...]
#         data:
#             perception_data1: initial_value1
#             perception_data2: initial_value2

rotation_activity:
    target_memory: 'social_events_memory'
    events: ['top_up', 'left_up', 'right_up', 'front_up', 'rear_up', 'bottom_up']
    data:
        rotation_x: 0
        rotation_y: 0
        rotation_z: 0

